include/retdec/fileformat/file_format/file_format.h

```diff
@@ -67,6 +81,7 @@ class FileFormat : public retdec::utils::ByteValueStorage, private retdec::utils
 		PdbInfo *pdbInfo;                                                 ///< information about related PDB debug file
 		CertificateTable *certificateTable;                               ///< table of certificates
 		Format fileFormat;                                                ///< format of input file
+		LoaderErrorInfo _ldrErrInfo;									  ///< loader error (e.g. Windows loader error for PE files)
```

Comment:
There are tabs in-between name of the attribute and comment. Please use spaces here. We use tabs only for indentation.

Reply:
Fixed in commit cab945158749486bd6a92ced84237678963a1291